components/brave_new_tab_ui/containers/newTab/settings/cards.tsx

```diff
@@ -58,7 +62,7 @@ const ToggleButton = ({ on, toggleFunc, float }: { on: boolean, toggleFunc: any,
   </StyledButton>
 }
 
-function CardSettings({ toggleShowBraveTalk, showBraveTalk, braveTalkSupported, toggleShowRewards, showRewards, braveRewardsSupported, toggleCards, cardsHidden }: Props) {
+function CardSettings({ toggleShowBraveTalk, showBraveTalk, braveTalkSupported, toggleShowBraveVPN, showBraveVPN, braveVPNSupported, toggleShowRewards, showRewards, braveRewardsSupported, toggleCards, cardsHidden }: Props) {
```

Comment:
this is why I generally prefer to avoid prop drilling in favor of accessing the data where its needed, either through direct access (for `loadTimeData`) or a hook (like `usePref`)

Reply:
All deleted.